### PR TITLE
fixes #8078 bug(nimbus): fix Takeaways edit button vertical padding

### LINKED ISSUE #8078: Takeaways "Edit" button vertical padding
1. Have a "Complete" experiment
2. On the Summary page, click on the "Archive" button on the lefthand side

When you archive the experiment, you can see the "Edit" button on the Takeaways section goes away. When it does, the vertical padding of the Takeaways section changes. This should be a consistent size whether or not the Edit button is present.

https://user-images.githubusercontent.com/43795363/208506481-b19ee2f7-b1e1-45ec-b056-76abcb0aa5aa.mov

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3012)

### PR DESCRIPTION
**Pull request by emmaz12**
Because

- The component containing the edit button's size is changed when the archive -button is clicked

This commit

- Allows the component to stay the same size when the archive button is clicked without changing
- Increases the margin for the column containing the Takeaway title and Edit button in order to solve the bug
- Updated app directory name to experimenter 

https://user-images.githubusercontent.com/117552772/223896504-96c8f7cd-c774-4082-8ac6-157a92728450.mov

### CODE REVIEW

**Comment by emmaz12** [comment]
@yashikakhurana Created new PR with correct root directory (changed from app to experimenter)

**Review by yashikakhurana — CHANGES_REQUESTED** [review]

File: experimenter/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx [hunk 1]
```diff
@@ -125,7 +125,7 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
               <th>Population %</th>
               <td data-testid="experiment-population">
                 {experiment.populationPercent ? (
-                  `${experiment.populationPercent}%`
+                  `${Number(experiment.populationPercent).toFixed(1)}%`
```
[the remark this answers is not in the document]
Comment (emmaz12) [reply experimenter/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx:128]: @yashikakhurana Sorry, meant to create a new PR, accidently commited to wrong branch. Will amend. Thanks!

**Comment by emmaz12** [comment]
@yashikakhurana This PR should be fixed now, created another PR to fix the population percent bug. Thanks! Sorry about that.

**Review by yashikakhurana — APPROVED** [review]
thank you @emmaz12 perfect solution, great job 👏

**Pull request merged**